Contributing.md

```diff
@@ -98,15 +98,15 @@ Once we've discussed your changes and you've got your code ready, make sure that
         $ make lint
         luacheck -q lua
         Total: 0 warnings / 0 errors in 74 files
-        ./utils/lj-releng lua/*.lua \
-            lua/apisix/*.lua \
-            lua/apisix/admin/*.lua \
-            lua/apisix/core/*.lua \
-            lua/apisix/http/*.lua \
-            lua/apisix/http/router/*.lua \
-            lua/apisix/plugins/*.lua \
-            lua/apisix/plugins/grpc-transcode/*.lua \
-            lua/apisix/plugins/limit-count/*.lua > \
+        ./utils/lj-releng \
```

Comment:
We need add `apisix.lua` in project root directory.

Reply:
moved `apisix.lua` to `apisix/init.lua`